mojo/core/broker_castanets.cc

```diff
@@ -415,7 +415,7 @@ void BrokerCastanets::AddSyncFence(const base::UnguessableToken& guid,
                                    uint32_t fence_id) {
   fence_queue_->AddFence(guid, fence_id);
 }
-
+#if defined(CASTANETS)
```

Comment:
I think we don't need the guard in this file, broker_castanets.cc is compiled only when castanets is enabled.

Reply:
Hmm, that's why I originally didn't add the flag, but I forgot.
Thank you for your review.

Reply:
Done.